parser/parser.y

```diff
@@ -10067,6 +10067,10 @@ SetStmt:
 	{
 		$$ = &ast.SetSessionStatesStmt{SessionStates: $3}
 	}
+|	"SET" "RESOURCE" "GROUP" ResourceGroupName
```

Comment:
please also provide the sql command like `show current resource group` to show the resource group for current session.

Reply:
done